.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,30 @@
+name: NodeJS
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 13.x]
```

Comment:
and actions/setup-node seems to be never called with those version in the below steps